moonwalking/blocks/eth_generic.py

```diff
@@ -209,3 +154,45 @@ def get_method_signature(method_name):
         elif to_string:
             return decode_abi(['string'], HexBytes(result))[0].decode()
         return result
+
+    async def send_eth(self, priv, addrs):
+        tx = await EthereumGeneric.build_tx(self, priv, addrs)
```

Comment:
I'm still not sure why we'd want this. Seems like having it is really just repeating the base implementation